tests/console/yast2_xinetd.pm

```diff
@@ -27,7 +27,11 @@ sub run {
     script_run("yast2 xinetd; echo yast2-xinetd-status-\$? > /dev/$serialdev", 0);
 
     # check xinetd network configuration got started
-    assert_screen 'yast2_xinetd_startup', 90;
+    assert_screen([qw(yast2_xinetd_startup yast2_xinetd_core-dumped)]), 90;
+    if (match_has_tag('yast2_xinetd_core-dumped')) {
+        # softfail when yast2 crashed and throws core dumped message
+        record_soft_failure "bsc#1049433";
+    }
```

Comment:
You are not doing anything special when you match that tag. Why not just create a workaround needle with the tag "yast2_xinetd_startup" referencing the bug id?